pkg/pillar/types/zedroutertypes.go

```diff
@@ -2889,3 +2913,59 @@ func (status OnboardingStatus) LogDelete(logBase *base.LogObject) {
 func (status OnboardingStatus) LogKey() string {
 	return string(base.OnboardingStatusLogType) + "-" + status.Key()
 }
+
+// NetworkInstanceIPAddrMapInfo :
+type NetworkInstanceIPAddrMapInfo struct {
+	// Collection of address assignments; from MAC address to IP address
+	// and IP Address to MAC
+	IPAssignments  map[string]net.IP
+	MacAssignments map[string]string
+}
+
+// NetworkInstanceAppNumMapInfo :
+type NetworkInstanceAppNumMapInfo struct {
+	DhcpAppNumCount int
```

Comment:
This field we can save in networkInstanceStatus (as some new DhcpRange.Size field)